llvm/lib/Target/AMDGPU/SIWholeQuadMode.cpp

```diff
@@ -583,40 +572,33 @@ char SIWholeQuadMode::scanInstructions(MachineFunction &MF,
         }
         GlobalFlags |= StateExact;
         III.Disabled = StateWQM | StateStrict;
-        continue;
-      } else {
-        if (Opcode == AMDGPU::SI_PS_LIVE || Opcode == AMDGPU::SI_LIVE_MASK) {
```

Comment:
The main motivation was to move the if-else chain that starts here "up" into the surrounding if-else chain, because I found it confusing.